test/strategy/yearn/YearnStrategy.spec.ts

```diff
@@ -145,6 +145,36 @@ describe('YearnStrategy', () => {
     });
   });
 
+  describe('#transferOwnership', () => {
+    it('can only be called by the current owner', async () => {
+      await expect(
+        strategy.connect(alice).transferOwnership(alice.address),
+      ).to.be.revertedWith('Ownable: caller is not the owner');
+    });
+
+    it('reverts if new owner is address(0)', async () => {
+      await expect(
+        strategy.connect(owner).transferOwnership(constants.AddressZero),
+      ).to.be.revertedWith('StrategyOwnerCannotBe0Address');
+    });
+
+    it('changes ownership to the new owner', async () => {
+      await strategy.connect(owner).transferOwnership(alice.address);
```

Comment:
For consistency I think here worth asserting who the owner was before also, and that the initial owner is no longer the owner after the call

Reply:
We have a test before this one that checks that only the owner can call transferOwnership method, IMO this would be redundant since the code reads clearly. I'm striving to have simple tests with no unnecessary code but with clear intent

Reply:
yap, this call would fail if "owner" is not the owner or if "alice" is the same as "owner"